grawlix-cms/_admin/book.page-edit.php

```diff
@@ -692,11 +692,76 @@
 	$headline_output = $page-> pageInfo['blog_title'];
 }
 
+//Script for adding HTML tags, just as in the Create Page file.
 $blog_output = <<<EOL
 <label for="blog_headline">Headline</label>
 <input type="text" name="blog_headline" id="blog_headline" value="$headline_output"/>
 <label for="blog_post">Post</label>
-<textarea name="blog_post" id="blog_post" rows="7">{$page-> pageInfo['blog_post']}</textarea>
+<script language="javascript" type="text/javascript">
+
+function addText(Text,Message)
+{
+  var obj = document.form.blog_post;
+
+  obj.focus();
+
+  if (document.selection && document.selection.createRange)  // Internet Explorer
+  {
+sel = document.selection.createRange();
+if (sel.parentElement() == obj)  sel.text = Text;
+  }
+
```

Comment:
While I'm fine with your style of putting `{` on a separate line, I think leaving empty lines between a `}` and the subsequent `else is` makes the code harder to follow. Good spacing visually links together related code. An empty space after an `if() {}` block makes it look like there's no `else`, but there is.